benchmarks/burgers/burgers_diff.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# ========================================================================================
+# (C) (or copyright) 2023. Triad National Security, LLC. All rights reserved.
+#
+# This program was produced under U.S. Government contract 89233218CNA000001 for Los
+# Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+# for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+# in the program are reserved by Triad National Security, LLC, and the U.S. Department
+# of Energy/National Nuclear Security Administration. The Government is granted for
+# itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+# license in this material to reproduce, prepare derivative works, distribute copies to
+# the public, perform publicly and display publicly, and to permit others to do so.
+# ========================================================================================
+
+import numpy as np
+from argparse import ArgumentParser
+
+parser = ArgumentParser(
+    prog="burgers_diff.py",
+    description="Compute difference between two history solvers parthenon VIBE",
+)
+parser.add_argument("file1", type=str, help="First file in diff")
+parser.add_argument("file2", type=str, help="Second fiel in diff")
+parser.add_argument(
+    "-t", "--tolerance", type=float, default=1e-8, help="Relative tolerance for diff"
+)
+
+def get_rel_diff(d1, d2):
+    "Get relative difference between two numpy arrays"
+    return 2 * np.abs(d1 - d2) / (d1 + d2 + 1e-20)
```

Comment:
How about using some of the [numpy testing](https://numpy.org/doc/stable/reference/routines.testing.html) functions instead (which may be helpful wrt to floating point comparisons at different scales)?

Reply:
This looks cool---I wasn't aware of these. But they all seem to be asserts, whereas I would prefer to print an error message and the location in the array that is off. Is there functionality for that?

Reply:
There are also the non-testing/assert versions of most of the functionality, e.g., https://numpy.org/doc/stable/reference/generated/numpy.isclose.html

Reply:
Hmm this still doesn't quite meet needs I think, as it doesn't easily output this diff.